cli/utils.go

```diff
@@ -170,7 +176,7 @@ func constructVersionInfo(version string) VersionInfo {
 			Major:  sv.Major,
 			Minor:  sv.Minor,
 			Patch:  sv.Patch,
-			Commit: pres[1],
+			Commit: pres,
```

Comment:
This seems the likely cause of failure. Maybe the commit is missing for some reason?